docs/docs/contributors.md

```diff
@@ -0,0 +1,27 @@
+---

```

Comment:
The contributors get updated automatically by the `@all-contributors` bot. This document will get out of sync :/